### ISSUE TITLE
Formatting `SELECT random()` throws an exception

### ISSUE THREAD

**Issue by jawj** [darold/pgFormatter#183]
A simple test case for this valid query:

```
echo 'SELECT random()' | perl ./lib/pgFormatter/pg_format

Uncaught exception: Deadly warning: Use of uninitialized value in quotemeta at [...]/lib/pgFormatter/Beautify.pm line 1350.
 at ./lib/pgFormatter/pg_format line 27.
[...]
```

**Comment by darold** [comment]
Thanks for the report, fixed in commit 9c2a100.

**Comment by jawj** [comment]
Quick work! Thanks.
